app/models/journeyDomain/incident/equipment/EquipmentDomain.scala

```diff
@@ -87,13 +87,13 @@ object EquipmentDomain {
       }
 
     IncidentCodePage(incidentIndex).reader.flatMap {
-      case TransferredToAnotherTransport | UnexpectedlyChanged =>
+      case x if x.code == "3" || x.code == "6" =>
```

Comment:
Again, could map over the code to make the pattern matching simpler. And can use `Constants`